go.mod

```diff
@@ -14,9 +14,6 @@ require (
 	github.com/giantswarm/microkit v1.0.0
 	github.com/giantswarm/micrologger v0.6.0
 	github.com/giantswarm/operatorkit/v6 v6.1.0
-	github.com/giantswarm/release-operator/v2 v2.5.0
-	github.com/giantswarm/resource/v4 v4.0.0
```

Comment:
I've rewritten the `app` resource to use the simple interface not the CRUD interface. This means we can deprecate the resource library as its only used by cluster-operator.

This PR deletes the app resource and the next will use the simple interface. So the diff is a bit cleaner.